packages/cli/src/utils/typescript.js

```diff
@@ -1,34 +1,16 @@
 import path from 'path'
-import { existsSync } from 'fs'
-import chalk from 'chalk'
 import consola from 'consola'
-import { warningBox } from './formatting'
-
-const dependencyNotFoundMessage =
-`Please install @nuxt/typescript and rerun the command
-
-${chalk.bold('Using yarn')}
-yarn add -D @nuxt/typescript
-
-${chalk.bold('Using npm')}
-npm install -D @nuxt/typescript`
 
 export async function detectAndSetupTypeScriptSupport(rootDir, options = {}) {
-  const tsConfigPath = path.resolve(rootDir, 'tsconfig.json')
-
-  if (!existsSync(tsConfigPath)) {
-    return false
-  }
-
-  consola.info(`${chalk.bold.blue('tsconfig.json')} found, enabling TypeScript runtime support`)
-
   try {
     const { setup } = require('@nuxt/typescript')
-    await setup(tsConfigPath, options)
+
+    consola.info('Initializing TypeScript Support')
```

Comment:
Can we show a message like "Creating /full/path/to/tsconfig.json" ?

```js
const tsConfigPath = path.resolve()
consola.info
await setup
```

Reply:
What about just replace `tsconfig.json` by full path here ? : https://github.com/nuxt/nuxt.js/pull/5367/commits/fe3ecd63588390fdea30ba231c03a8a3eed4f78a#diff-8404199e6ca4961076ae35d7afeb980aR48

Reply:
Yeah. Cool.